### PR TITLE
Added GitHub build

### PR DESCRIPTION
**Pull request by droyad**

### CODE REVIEW

File: src/Samples/FirebirdSampleApplication/FirebirdSampleApplication.csproj [hunk 1]
```diff
@@ -4,6 +4,8 @@
     <RootNamespace>FirebirdSampleApplication</RootNamespace>
     <AssemblyName>FirebirdSampleApplication</AssemblyName>
     <OutputType>Exe</OutputType>
+    <NoWarn>$(NoWarn);NETSDK1138</NoWarn> <!-- Purposefully leaving an old version of netcoreapp to ensure we have compatibility. This never gets published -->
+    <IsPackable>false</IsPackable>
```
Comment (droyad) [inline src/Samples/FirebirdSampleApplication/FirebirdSampleApplication.csproj:8]: Stop packaging these sample apps

File: install-sql-ce.ps1 [hunk 2]
```diff
@@ -0,0 +1,5 @@
+$url = "https://download.microsoft.com/download/F/F/D/FFDF76E3-9E55-41DA-A750-1798B971936C/ENU/SSCERuntime_x64-ENU.exe"
+$WebClient = New-Object System.Net.WebClient
+$WebClient.DownloadFile($url ,"$env:TEMP/SSCERuntime_x64-ENU.exe");
+
+Start-Process "$env:TEMP/SSCERuntime_x64-ENU.exe" -argumentlist "/i /quiet" -wait
```
Comment (droyad) [inline install-sql-ce.ps1:5]: Used by the build process

File: src/dbup-firebird/dbup-firebird.csproj [hunk 3]
```diff
@@ -11,9 +11,9 @@
     <PublicSign Condition=" '$(OS)' != 'Windows_NT' ">true</PublicSign>
     <PackageId>dbup-firebird</PackageId>
     <PackageReleaseNotes>https://github.com/DbUp/DbUp/releases</PackageReleaseNotes>
-    <PackageIconUrl>https://raw.github.com/DbUp/DbUp/master/src/Information/dbup-icon.png</PackageIconUrl>
+    <PackageIconUrl>../../dbup-icon.png</PackageIconUrl>
     <PackageProjectUrl>https://dbup.github.io</PackageProjectUrl>
-    <PackageLicenseUrl>http://www.opensource.org/licenses/mit-license.php</PackageLicenseUrl>
+    <PackageLicenseExpression>MIT</PackageLicenseExpression>
```
Comment (droyad) [inline src/dbup-firebird/dbup-firebird.csproj:16]: The *Url properties are deprecated

File: build.ps1 [hunk 4]
```diff
@@ -1,145 +1,13 @@
...
-.SYNOPSIS
-This is a Powershell script to bootstrap a Cake build.
+$env:DOTNET_SKIP_FIRST_TIME_EXPERIENCE = '1'
+$env:DOTNET_CLI_TELEMETRY_OPTOUT = '1'
+$env:DOTNET_NOLOGO = '1'
 
-.DESCRIPTION
-This Powershell script will download NuGet if missing, restore NuGet tools (including Cake)
-and execute your Cake build script with the parameters you provide.
+dotnet tool restore
```
Comment (AdrianJSClark) [inline build.ps1:9]: 😍 Love replacing all this PowerShell with basically 4 lines.

**Review by AdrianJSClark — APPROVED** [review]
Looks good! Love it.

**Pull request merged**